RHEL/7/input/oval/enable_x11_forwarding.xml

```diff
@@ -0,0 +1,29 @@
+<def-group>
+  <definition class="compliance" id="enable_x11_forwarding" version="1">
+    <metadata>
+      <title>Enable X11 Forwarding</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 7</platform>
+      </affected>
+      <description>Enable X11Forwarding to encrypt X11 remote connections over SSH.</description>
+      <reference source="galford" ref_id="20160410" ref_url="test_attestation" />
+    </metadata>
+    <criteria comment="SSH is not being used or conditions are met"
+    operator="OR">
+      <extend_definition comment="sshd service is disabled"
+      definition_ref="service_sshd_disabled" />
```

Comment:
Taking my previous note back ```service_sshd_disabled``` is present in both OVAL-5.10 and OVAL-5.11 case for RHEL-7. Sorry. Pls ignore it.